README.md

```diff
@@ -62,7 +62,7 @@ $ export NODE_ENV=dev
 $ export DEBUG='ccd-case-activity-api:*'
 $ yarn start
 
-> ccd-case-activity-api@0.0.1 start /Users/fatiho/Documents/hmcts/ws-webstorm/ccd-case-activity-api
+> ccd-case-activity-api@0.0.2 start /Users/fatiho/Documents/hmcts/ws-webstorm/ccd-case-activity-api
 > node ./bin/www
```

Comment:
What does notation > ccd-case-activity-api@0.0.2 mean?

Reply:
I think it is a copy and paste from fatih's shell prompt :)

Reply:
I think is a copy and paste of fatih's shell prompt when he wrote the readme